homeassistant/components/homematic.py

```diff
@@ -276,7 +276,6 @@ def setup(hass, config):
     # Stops server when Homeassistant is shutting down
     hass.bus.listen_once(
         EVENT_HOMEASSISTANT_STOP, hass.data[DATA_HOMEMATIC].stop)
-    hass.config.components.append(DOMAIN)
```

Comment:
Yeah, it's the responsibility of bootstrap to do this. Not sure how it ended up here